modules/microshift-backing-up-manually.adoc

```diff
@@ -4,26 +4,35 @@
 
 :_content-type: PROCEDURE
 [id="microshift-backing-up-manually_{context}"]
-= Backing up {product-title} data manually
+= Backing up {microshift-short} data manually
 
-You can back up {product-title} data manually at any time. Data backups are advised before any system updates. Backups are created in the `/var/lib/microshift-backups` directory by default. You can also specify a directory by adding custom parameters to the required backup command.
+You can back up {microshift-short} data manually at any time. Data backups are advisable before any system updates. Automated backups are created in the `/var/lib/microshift-backups` directory. You can use this directory for manually backing up and restoring data by specifying it in each command. When you create a backup, you must use the entire file path for the output file.
 
 [NOTE]
 ====
-On `rpm-ostree` systems, {product-title} creates an automatic backup on every start and restores data automatically.
+On `rpm-ostree` systems, {microshift-short} creates an automatic backup on every start and restores data automatically.
```

Comment:
The condition for automated restore is quite specific, should we specify it further or remove it altogether since this sections is about backing up?

Reply:
I removed it. We have it in the parent assembly, no need to mention it twice.